ui/tests/integration/components/masked-input-test.js

```diff
@@ -54,36 +52,12 @@ module('Integration | Component | masked input', function(hooks) {
     assert.notOk(component.copyButtonIsPresent);
   });
 
```

Comment:
Should we add a test for `Does not unmask text on focus`?

Reply:
Good call.  I added a test to cover this.